src/utils.js

```diff
@@ -235,3 +235,30 @@ export function apiToMessage(string) {
 export function isBrowserNamespace(string) {
   return ['browser', 'chrome'].includes(string);
 }
+
+
+export function parseCspPolicy(policy) {
```

Comment:
Well, none of them have been touched in recent years so I wouldn't be surprised if they're even maintained. If you're okay with it I feel more confident having that code in our code-base since we have to validate those rules on a relatively low level anyway.